src/Standards/PSR2/Tests/ControlStructures/ControlStructureSpacingUnitTest.inc

```diff
@@ -45,3 +45,49 @@ foreach (  $something as $blah => $that  ) {}
 // @codingStandardsChangeSetting PSR2.ControlStructures.ControlStructureSpacing requiredSpacesBeforeClose 0
 
 $binary = b"binary string";
+
+// @codingStandardsChangeSetting PSR2.ControlStructures.ControlStructureSpacing newlineAllowedAfterOpen false
+// false is default
+
+if (
+    $priceWithoutVatFrom === null ||
+    $priceWithoutVatTo === null ||
+    $priceWithVatFrom === null ||
+    $priceWithVatTo === null
+) {
+    return new PriceFilterConfig(0, 0, 0, 0);
+}
+
+try {
+    throw new \Exception();
+} catch (
+    \MyNamespace\FullyQualified\Long\Make\Exception\MakeNotFoundBySlugException |
+    \MyNamespace\FullyQualified\Long\Model\Exception\ModelNotFoundBySlugAndMakeException |
+    \MyNamespace\FullyQualified\Long\ModelLine\Exception\ModelLineNotFoundBySlugAndModelException
+    $e
+) {
+    // ...
+}
+
+// @codingStandardsChangeSetting PSR2.ControlStructures.ControlStructureSpacing newlineAllowedAfterOpen true
+// newline after opening bracket should be OK here
+
+if (
+    $priceWithoutVatFrom === null ||
+    $priceWithoutVatTo === null ||
+    $priceWithVatFrom === null ||
+    $priceWithVatTo === null
+) {
+    return new PriceFilterConfig(0, 0, 0, 0);
+}
+
+try {
+    throw new \Exception();
+} catch (
+    \MyNamespace\FullyQualified\Long\Make\Exception\MakeNotFoundBySlugException |
+    \MyNamespace\FullyQualified\Long\Model\Exception\ModelNotFoundBySlugAndMakeException |
+    \MyNamespace\FullyQualified\Long\ModelLine\Exception\ModelLineNotFoundBySlugAndModelException
+    $e
+) {
+    // ...
+}
```

Comment:
Please reset the property to the default after the tests targeting it, to make sure it will not have an impact on additional unit tests being added later.

Reply:
fixed